Base/usr/share/man/man1/test.md

```diff
@@ -0,0 +1,89 @@
+## Name
+
+test - checks files and compare values
+
+## Synopsis
+
+```**sh
+$ test EXPRESSION
+```
+
+## Description
+
+`test` takes a given expression and sets the exit code according to its truthiness, 0 if true, 1 if false.
+An omitted expression defaults to false, and an unexpected error causes an exit code of 126.
+
+## Expressions
+
+The expression can take any of the following forms:
+
+### Grouping
+
+* `( <expression> )` (grouping)
+
+### Boolean operations
+
+* `! <expression>` (negation)
+* `<expression> -a <expression>` (and)
+* `<expression> -o <expression>` (or)
+
+### String comparison
+
+* `-n <string>` (non-empty)
+* `-z <string>` (empty)
+* `<string> = <string>` (equality)
+* `<string> != <string>` (unequality)
+* `<string>` (non-empty, if the only expression specified)
+
+### Integer comparison
+
+* `<integer> -eq <integer>` (equality)
+* `<integer> -ne <integer>` (unequality)
+* `<integer> -lt <integer>` (less-than)
+* `<integer> -gt <integer>` (greater-than)
+* `<integer> -le <integer>` (less-than-or-equal)
+* `<integer> -ge <integer>` (greater-than-or-equal)
+
+### File comparison
+
+* `<file> -ef <file>` (same file - same device and inode number)
+* `<file> -nt <file>` (newer-than - checks modification date)
+* `<file> -ot <file>` (older-than - checks modification date)
+
+### File type checks
+
+* `-b <file>` (is block device?)
+* `-c <file>` (is character device?)
+* `-f <file>` (is regular file?)
+* `-d <file>` (is directory?)
+* `-p <file>` (is pipe?)
+* `-S <file>` (is socket?)
+* `-h <file>`, `-L <file>` (is symbolic link?)
+
+### File permission checks
+
+* `-r <file>` (is readable?)
```

Comment:
Perhaps these can be rewritten without the parens and question marks?

E.g. `-r <file>` — whether the file is readable